Fabric.Authorization.Domain/Services/EDWAdminRoleSyncService.cs

```diff
@@ -66,14 +78,14 @@ private async Task<bool> IsUserASuperAdmin(User user)
             // if a group is a special Super Admin Group then return true
             // if not, then look on each group to see if there is a "super admin role"
             // finally, look on each group's children groups to see if there is a "super admin role"
-            foreach (var customGroup in user.Groups)
+            foreach (var customGroup in user.Groups.Where(p=> !p.IsDeleted))
```

Comment:
Are we returning deleted groups in the query that retrieves this user? That probably needs to be fixed at some point if so.